gradle/libs.versions.toml

```diff
@@ -0,0 +1,65 @@
+[versions]
+arcgisVersion = "200.4.0-4085"
```

Comment:
Give a better name for this, `arcgisMapsKotlin`, since all other libs omit the `version` appendix.